.make/test.mk

```diff
@@ -147,6 +147,7 @@ test-unit-with-coverage: prebuild-check clean-coverage-unit $(COV_PATH_UNIT)
 test-unit: prebuild-check $(SOURCES)
 	$(call log-info,"Running test: $@")
 	$(eval TEST_PACKAGES:=$(shell go list ./... | grep -v $(ALL_PKGS_EXCLUDE_PATTERN)))
+	go clean -cache
```

Comment:
Why do you want to clean cache? Adding `-p 1` flag slows down tests. So, cache is actually quite useful when running tests locally. You can always clean cache manually between test runs if you want.

Reply:
right, let me remove that. I just wanted to make sure that there was no cache when running the integration tests to reproduce the "parallel run" failures, but I can surely remove that one now.

Reply:
done in dc31c0f